Mail/Helpers/AppAssembly.swift

```diff
@@ -121,6 +121,9 @@ enum ApplicationAssembly {
             },
             Factory(type: ConfigWebServer.self) { _, _ in
                 ConfigWebServer()
+            },
+            Factory(type: AppLaunchCounter.self) { _, _ in
```

Comment:
There is another Assembly for the share extension. 
I think the Extension will work without knowing about the `AppLaunchCounter` since it is used only in the `SplitView` and `MailApp`.

We should verify the share extension still work to be sure.

Reply:
Not working.